src/config/production.ts

```diff
@@ -0,0 +1,11 @@
+import { IConfig } from './types';
+
+const config: IConfig = {
+  env: 'production',
+  productName: 'Quiz Master',
+  productUrl: 'https://quizmaster.com',
```

Comment:
quizmaster.vercel.app?
Are you planning to buy this domain?

Reply:
can't we change the domain in `vercel` app. We can't use 
https://quizmaster-pvmuay9az-rajatkantinandi.vercel.app/  for production

Reply:
Yes we can. The domain here should be quizmaster.vercel.app

Reply:
cool